Jenkinsfile

```diff
@@ -84,11 +86,22 @@ pipeline {
             environment {
                 KOJI_KEYTAB = credentials('fedora-keytab')
                 KRB_PRINCIPAL = 'bpeck/jenkins-continuous-infra.apps.ci.centos.org@FEDORAPROJECT.ORG'
+                // TODO: What's the actual host?
+                VALKEY_HOST = "localhost"
             }
 
             steps {
                 script {
-                    timeout(time: 600, unit: 'MINUTES') {
+                    lock("check-concurrent-builds") {
+                        def jobname = env.JOB_NAME
```

Comment:
The `JOB_NAME` I am not 100% about. Don't know if it `dist-git-build-pipepline` or something else that would not catch other PR